src/common/page.rs

```diff
@@ -51,13 +51,26 @@ pub struct Chunk {
   pub size: usize, // pub owned: false
 }
 
-
 impl fmt::Display for Chunk {
   fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
     write!(f, "Chunk {{ptr:{}, size:{}}}", self.ptr as usize, self.size)
   }
 }
 
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct Run {
+  pub length: *const u8,
+  pub value: *const u8,
```

Comment:
I thought this code is cpp. That is I saw it as like ``*uint8_t``. I was blind. Because this code is in Rust, ``*const u8`` is right. ``*const u8`` in Rust is conventionally used like void in C or C++.